src/gameskyrimse.cpp

```diff
@@ -198,8 +198,7 @@ MOBase::VersionInfo GameSkyrimSE::version() const
 
 QList<PluginSetting> GameSkyrimSE::settings() const
 {
-  return {PluginSetting("enderal_downloads", "allow Enderal and Enderal SE downloads",
-                        QVariant(false))};
+return {}; //Must return something, otherwise linker erros smh
```

Comment:
You obviously needs to return an empty list in a function that as a `QList` return type so that comment is not really useful.